ee/models/rbac/access_control.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+
+from posthog.models.utils import UUIDModel
+
+
+class AccessControl(UUIDModel):
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["resource", "resource_id", "team", "organization_member", "role"],
+                name="unique resource per target",
+            )
+        ]
+
+    team: models.ForeignKey = models.ForeignKey(
+        "posthog.Team",
+        on_delete=models.CASCADE,
+        related_name="access_controls",
+        related_query_name="access_controls",
+    )
+
+    # Configuration of what we are accessing
+    access_level: models.CharField = models.CharField(max_length=32)
```

Comment:
I suppose we have the available access levels defined somewhere, so could we include `choices` in this field to have autovalidation, etc?

Reply:
We don't want to do db level validation to avoid migrations needed so it'll all be in the serializer. Coming soon :)